src/components/Button/Button.js

```diff
@@ -18,16 +18,23 @@ const { defaultProps, propTypes } = CarbonButton;
 
 // TODO: V3 - Remove deprecated props `largeText`.
 
-const Button = ({ className, largeText, size, ...other }) => {
+const Button = ({ className, largeText, size, kind, ...other }) => {
   const isSize = value => size === value;
   const isLarge = isSize('large') || largeText || isSize('lg') || isSize('xlg');
 
+  const isKind = value => kind === value;
+  const isGhostDanger = isKind('ghost-danger');
+
+  console.log(!isGhostDanger ? kind : null);
+
   return (
     <CarbonButton
       className={classnames(namespace, className, {
         [`${namespace}--large`]: isLarge,
+        [`${namespace}--ghost-danger`]: isGhostDanger,
       })}
       size={!isLarge ? size : null}
+      kind={!isGhostDanger ? kind : 'ghost'}
```

Comment:
Just checking -- are you keeping the `kind='ghost'` in this case to get the Carbon ghost button classes?

Reply:
Yes, as that is how the implementation worked in the Tearsheet component.

I am not too familiar with the carbon system, so there might be a more efficient and future proof way of doing this, if so, do let me know.